sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/ExternalTypescriptSensor.java

```diff
@@ -328,6 +343,37 @@ private void saveHighlights(SensorContext sensorContext, Highlight[] highlights,
     highlighting.save();
   }
 
+
+  @Nullable
+  private static File getTypescriptLocation(File topDir) {
```

Comment:
@vilchik-elena what about extracting this in its own class and writing a dedicated unit test?